model/relabel/relabel_test.go

```diff
@@ -758,3 +763,41 @@ func BenchmarkRelabel(b *testing.B) {
 		})
 	}
 }
+
+func TestConfig_UnmarshalYAML(t *testing.T) {
+	type fields struct {
+		Source string
+	}
+
+	tests := []struct {
+		name    string
+		fields  fields
+		wantErr bool
+	}{
+		{
+			name: "keepequal",
+			fields: fields{
+				Source: `
+action: keepequal
+regex: (.*)
```

Comment:
I expect this config to fail as the value is set explicitly.

Reply:
You may got this value from a normal yaml marshal, i expect this not to fail

Reply:
Otherwise, we need to implement MarshalYAML for Config